### DISCUSSION TITLE
Multiple layers of blank modal when navigate to a page or search on a resource

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#14954] [Help]
### Package

Panel builder

### Package Version

3.2.124

### How can we help you?

Weird behavior when I'm on a filament resource index page

Multiple layers of blank modals that covers the whole screen will pop up if I try to 

1. Navigate to another page
2. Type something in the search bar

And nothing comes up in the `laravel.log` not even sure how to start troubleshooting..

## Update 2024-12-03

To top it off, after I deploy this code to my staging and production environment, the issue no longer appears

See images below of my filament using white theme

### Before
![image](https://github.com/user-attachments/assets/a44cc142-8ad1-4eb2-89a2-48ceaeb240cf)

### After
- covered in multiple layers of black background modal, knew by using Chrome dev tools
![image](https://github.com/user-attachments/assets/6165c500-131e-404f-adc1-3e66b73627f5)

**Comment by leandrocfe** [comment]
What code are you using?

**Reply by bilogic** [reply]
Not sure what is being asked, I'm on a filament resource index page

**Reply by leandrocfe** [reply]
_a filament resource index page_ what does it mean?

**Reply by bilogic** [reply]
what do you call this page? https://demo.filamentphp.com/shop/customers

The problem appears if I search or navigate to other pages boxed in red
![image](https://github.com/user-attachments/assets/1725d906-23b9-4e15-98f6-17d9fdba11d4)

**Comment by untillnesss** [comment]
Upgrade your Filament version, by running `composer update`
I think there is a big change in Alpine JS.

**Reply by bilogic** [reply]
I run `composer update` multiple times a day, does not fix my issue.

**Reply by leandrocfe** [reply]
where are the blank modals?

**Reply by bilogic** [reply]
see my OP
